dotcom-rendering/src/web/components/Liveness.importable.tsx

```diff
@@ -284,13 +289,25 @@ export const Liveness = ({
 		}
 	}, [hasPinnedPost, onFirstPage, webURL]);
 
-	if (showToast) {
-		return (
+	if (toastRoot && showToast) {
+		/**
+		 * createPortal?
+		 *
+		 * We're using a portal here because of a the way Safari implements position: sticky. If
+		 * the element is rendered directly here as a child of the Island (its parent) then stickiness
+		 * won't work in Safari because this browser positions the element relative to the immediate
+		 * parent, whereas Chrome et al are more forgiving.
+		 *
+		 * By using a Portal we can insert the Toast as a sibling to the Island, which works around
+		 * Safari's behaviour.
+		 */
```

Comment:
```suggestion
		/**
		 * Why `createPortal`?
		 *
		 * We use a Portal because of a the way different browsers implement `position: sticky`.
		 * A stickily positioned element depends on its [containing block][] to determine
		 * when to become stuck.
		 *
		 * In Safari the containing block is set to the immediate parent
		 * whereas Chrome, Firefox and other browser are more forgiving.
		 *
		 * By using a Portal we can insert the Toast as a sibling to the Island, which works around
		 * Safari's behaviour.
		 *
		 * [containing block]: https://developer.mozilla.org/en-US/docs/Web/CSS/Containing_block#identifying_the_containing_block
		 */
```

Reply:
Thanks! 🙏